### PR TITLE
modifications in functions chapter

### PR DESCRIPTION
**Pull request by juans-ally**

### CODE REVIEW

**Comment by iteducacion** [comment]
send the updates in functions.. please tell me if is it good....

File: 08_functions/pizza.py [hunk 1]
```diff
@@ -7,5 +7,6 @@ def make_pizza(size, *toppings):
         print(f"- {topping}")
         if ( topping == 'napolitana' ):
             print('Me encanta la de napolitana!')
-
+        if ( topping == 'extra cheese' ):
+            print('mmmmm, te gusta la de queso extra!')
```
Comment (juans-ally) [inline 08_functions/pizza.py]: change the "," with ".... " please

it looks more beautiful

**Review by iteducacion — CHANGES_REQUESTED** [review]
wait for that change please....

**Comment by iteducacion** [comment]
Perfect! merge the updates in functions

**Pull request merged**